packages/service-library/src/servicelib/long_running_tasks/_task.py

```diff
@@ -126,7 +127,7 @@ async def _stale_tasks_monitor_worker(self) -> None:
                 )
 
     @staticmethod
-    def _create_task_id(task_name: TaskName) -> str:
+    def generate_task_id(task_name: TaskName) -> str:
```

Comment:
Q: why not `create`?  THOUGHT: IMO it is better to have a reduced and well defined number of verbs for function names, e.g. `get`, `set`, `new`/`create`, `pop`, `push`, `rm` ... and avoid using too many synonyms